README.md

```diff
@@ -10,12 +10,12 @@ all of the probes we expect to see in the appropriate places.
 
 ## Telemetry Integration
 
-There are two pings we are targeting for integration with this library:
+The two primary pings we are targeting for integration with this library are:
```

Comment:
In fact, we are targeting a [whole host of pings now](https://github.com/mozilla/mozilla-schema-generator/tree/master/mozilla_schema_generator/configs). We should probably list two generic ping types we're targeting:
- Common Pings (the main ping being an example, this is legacy telemetry)
- Glean Pings